docs-source/content/userguide/managing-fmw-domains/soa-suite/Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,126 @@
+# Weblogic Logging Exporter Setup to publish logs into Elasticsearch
+
+The WebLogic Logging Exporter adds a log event handler to WebLogic Server. 
+
+WebLogic Server logs can be pushed into Elastic Stack in Kubernetes directly, by using the Elasticsearch REST API. For more details refer to [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter) project.  
```

Comment:
@markxnelson, check me on this... logs get published into Elasticsearch... that's the log database.  Elastic Stack refers to the set of Elasticsearch, Logstash, and Kibana.  So, this should say that logs are pushed into Elastisearch.